### PR TITLE
Pr add miracle

### PR DESCRIPTION
**Pull request by wolf-yuan-6115**

### CODE REVIEW

File: blog/authors.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 kane:
   name: 凱恩Kane
-  title: YEE式機器龍作者
+  title: Yee式機器龍作者
```
Comment (Gary50613) [inline blog/authors.yml:3]: I want uppercase 🤔

**Review by Gary50613 — CHANGES_REQUESTED** [review]

File: blog/authors.yml (hunk 1 above)
Comment (wolf-yuan-6115) [reply blog/authors.yml:3]: == fine

Comment (Gary50613) [reply blog/authors.yml:3]: lol I'm so sorry

**Review by Gary50613 — APPROVED** [review]
LGTM, thanks for the contribute!

**Comment by wolf-yuan-6115** [comment]
Add webhook to discord

**Pull request merged**